src/components/common/markets/market-asset-price-chart.tsx

```
@@ -115,44 +154,89 @@ function PythLiveChart({ id }: PythLiveChartProps) {
-                Math.abs(priceEntity ? priceEntity.expo : 8)
+                BigInt(priceEntity.price),
+                Math.abs(priceEntity.expo)
               )
               const price = +formattedPrice.toFixed(6)
-              const latestPriceFeedEntity = priceFeed.getPriceNoOlderThan(60)
-              const currentTime = latestPriceFeedEntity
-                ? latestPriceFeedEntity.publishTime * 1000
-                : new Date().getTime()
-              // @ts-ignore
+
+              if (price <= 0 || !isFinite(price)) {
```

Biome (1.9.4)

[error] isFinite is unsafe. It attempts a type coercion. Use Number.isFinite instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isFinite instead.


(lint/suspicious/noGlobalIsFinite)